### PR TITLE
005 Deploying to AWS Back End

### PR DESCRIPTION
**Pull request by Adnanhoq**

### CODE REVIEW

**Review by DylanS-03 — APPROVED** [review]

File: AWSCommands [hunk 1]
```diff
@@ -0,0 +1,5 @@
+docker build --build-arg DB_VAR=${DB_VAR} --build-arg DB_USERNAME=${DB_USERNAME} --build-arg DB_PASSWORD=${DB_PASSWORD} --build-arg DB_HOST=${DB_HOST} --build-arg DB_NAME=${DB_NAME} -t myapp:0.1 .
+docker run -p 8080:8080 myapp:0.1
```
Comment (JennicaMistry) [inline AWSCommands]: Put this in the Readme.

**Review by JennicaMistry — CHANGES_REQUESTED** [review]

File: AWSCommands (hunk 1 above)
Comment (Adnanhoq) [reply AWSCommands]: thats a different ticket

Comment (JennicaMistry) [reply AWSCommands]: Nope

Comment (Adnanhoq) [reply AWSCommands]: Done by Tom

**Review by JennicaMistry — APPROVED** [review]

**Pull request merged**